### ISSUE TITLE
Add requestPermissions() to Android Test case "Testing App Permissions"

### ISSUE THREAD

**Issue by sushi2k** [OWASP/owasp-mstg#891]
https://developer.android.com/training/permissions/requesting

**Comment by commjoen** [comment]
Issue is assigned. Welcome back!

**Comment by commjoen** [comment]
I guess that this will require updating 0x05a (platform overview), 0x05f (platform interaction) and 0x05j

**Comment by B3nac** [comment]
Thank you, glad to be back! Can confirm that requestPermissions() has been added to 0x05a and see the perfect spot for "Testing App Permissions" in 0x05f. Working on an example now. 0x05j will be next.

**Comment by commjoen** [comment]
PR reviewed: the most important part here is updating the existing testcases which are about the permissions. Once that is done, then we can get another PR.

**Comment by commjoen** [comment]
(note: this issue touches upon https://github.com/OWASP/owasp-mstg/issues/956 as well)

**Comment by commjoen** [comment]
Thank you for implementing
